ostd/src/lib.rs

```diff
@@ -24,6 +24,9 @@
 #![no_std]
 #![warn(missing_docs)]
 
+// TODO - only for cfg test
```

Comment:
What's the right way to do this? You apparently can't just throw a `cfg(test)` at it.